etc/inc/system.inc

```diff
@@ -1311,6 +1311,8 @@ EOD;
 		// SSLv2/3 is deprecated, force use of TLS
 		$lighty_config .= "ssl.use-sslv2 = \"disable\"\n";
 		$lighty_config .= "ssl.use-sslv3 = \"disable\"\n";
+		$lighty_config .= "ssl.ec-curve = \"secp384r1\"\n";
+		$lighty_config .= "ssl.dh-file = \"/etc/dh-parameters.4096\"\n";
```

Comment:
Is it safe to use [`/etc/dh-parameters.4096`](https://raw.githubusercontent.com/pfsense/pfsense/master/etc/dh-parameters.4096) or should we generate another one for this purpose?